cargo-remove: add page (#10816)
tldr-pages/tldr: pages/common/cargo-remove.md

# cargo remove

> > Remove dependencies from a Cargo.toml manifest file.

Reviewer: ```suggestion
> Remove dependencies from a `Cargo.toml` manifest file.
```

> More information: <https://doc.rust-lang.org/cargo/commands/cargo-remove.html>.

> - Remove one or more dependency from the Cargo.toml manifest:

Reviewer: ```suggestion
- Remove one or more dependencies from the `Cargo.toml` manifest:
```

> - Remove one or more dependencies from the Cargo.toml manifest:

Reviewer: ```suggestion
- Remove one or more dependencies from the `Cargo.toml` manifest:
```

> `cargo remove`

Reviewer 2: I think we can't run bare `cargo remove` . We need to supply the dependency we want to remove as well.
```suggestion
`cargo remove {{dependency_name}}`
```

> - Remove as a build dependency:

Reviewer: ```suggestion
- Remove a build dependency:
```

> `cargo remove --build`

Reviewer 2: I think all `remove` commands do expect a `dependency_name`, so i would request to add it as appropriate.
I tried to run the `cargo remove` on a rust project and it expects a `dependency_name`.

Author: Okay my bad, I will fix it ASAP.

> - Remove as a dependency to the given target platform:

Reviewer: ```suggestion
- Remove a dependency to the given target platform:
```

`cargo remove --target {{target}}`

- Don't actually write to the manifest:

`cargo remove --dry-run`

> - Use verbose output:

Reviewer: ```suggestion
- Display verbose output:
```

`cargo remove --verbose`

- Do not print Cargo log message:

`cargo remove --quiet`

> - Package to remove from:
> 
> `cargo remove {{-p|--package}}  {{spec...}}`

Reviewer: ```suggestion
- Specify [p]ackage to remove from:

`cargo remove --package  {{specification}}`
```

Currently, we use mnemonics interchangably to highlight long and short options in examples.

> `cargo remove --package  {{specification}}`

Reviewer: ```suggestion
`cargo remove --package {{specification}}`
```

> - Print help information:

Reviewer: ```suggestion
- Display help:
```

# cargo remove

> Remove dependencies from a Cargo.toml manifest file.
> More information: <https://doc.rust-lang.org/cargo/commands/cargo-remove.html>

> - Remove one or more depndencies from the Cargo.toml manifest.

Reviewer 3: Typo: depndencies

`cargo remove`

- Remove as a development dependency.

`cargo remove --dev`

- Remove as a build dependency.

`cargo remove --build`

- Remove as a dependency to the given target platform.

`cargo remove --target {{target}}`

- Don't actually write to the manifest.

`cargo remove --dry-run`

> - Use verbrose output.

Reviewer 3: Typo: verbrose
